server/src/main/java/org/opensearch/common/settings/IndexScopedSettings.java

```diff
@@ -252,7 +252,6 @@ public IndexScopedSettings copy(Settings settings, IndexMetadata metadata) {
     }
 
     public boolean registerSetting(Setting<?> setting) {
```

Comment:
I think you don't need override anymore after https://github.com/opensearch-project/OpenSearch/pull/5362/files#diff-edf64b791aed5932da1f53fa264e42e98bd473f585ceed12892b66cda79c3fd7R147 ?

Reply:
@reta As in we can just delete the method entirely?